### PR TITLE
Add PR Template

### PR DESCRIPTION
**Pull request by jmaeng72**
I believe that having a standard PR template will help clarify PRs for reviewers and help developers go through testing processes

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+### What areas of the application does this impact?
+
+List impacted areas.
+
+# Testing
+
+### Performance Test Results
+
+Please put in details of performance results from testing in WL
```
Comment (jmaeng72) [inline .github/pull_request_template.md:19]: If we are uncomfortable putting performance results, then we can add this as a checklist instead. Please LMK what you think

Comment (isja17) [inline .github/pull_request_template.md:19]: we shouldn't require WL runs to be ran pre-pr. Most tickets won't even need WL runs.

Comment (jmaeng72) [reply .github/pull_request_template.md:19]: That makes sense, I can remove this

**Review by isja17 — APPROVED** [review]

**Review by eudoroolivares2016 — APPROVED** [review]
Great change

**Pull request merged**